tests/unit/src/services/birthDate.spec.js

```diff
@@ -0,0 +1,19 @@
+import { buildImportWalletBirthdate } from '@/services/birthDate'
+
+describe.only('birthDate', () => {
+  it('return 30 days less than the provided date', ()=> {
+    const date = new Date('2021-10-31')
+
+    const birthDate = buildImportWalletBirthdate(date)
+
+    expect(birthDate).toBe(675119) 
```

Comment:
Where does this number come from?

Reply:
The epoch for 2021-10-31. I'll add a comment